jiant/preprocess.py

```diff
@@ -52,7 +52,8 @@
 from jiant.tasks import REGISTRY as TASKS_REGISTRY
 from jiant.tasks.seq2seq import Seq2SeqTask
 from jiant.tasks.tasks import SequenceGenerationTask
-from jiant.utils import config, serialize, utils
+from jiant.utils import config, serialize, utils, options
```

Comment:
delete the `options` import ?

Reply:
You still need it for the build_tasks function